ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,493 @@
+---
+eip: xxxx
```

Comment:
```suggestion
eip: 8027
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.